packages/lodestar-params/src/beaconParams.ts

```diff
@@ -13,6 +14,7 @@ const ByteVector20 = new ByteVectorType({length: 20});
 export const BeaconParams = new ContainerType<IBeaconParams>({
   fields: {
     phase1: Phase1Params,
+    lightclient: LightClientParamsSSZ,
```

Comment:
should be `lightClient` for consistency

Reply:
Actually, reading thru the eth2 spec, it seems that Lightclient is one word (not two).

So should be `LightClient` => `Lightclient`, `lightClient` => `lightclient` everywhere for consistency.

Reply:
I dont think they can agree but yeah lets stick to the second one